modules/virt-metro-dr-odf.adoc

```diff
@@ -6,14 +6,11 @@
 [id="metro-dr-odf_{context}"]
 = Metro-DR for {rh-storage-first}
 
-{VirtProductName} supports the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.14/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for {rh-storage}], which provides two-way synchronous data replication between managed {VirtProductName} clusters installed on primary and secondary sites. This solution combines {rh-rhacm-first}, Red{nbsp}Hat Ceph Storage, and {rh-storage} components.
+{VirtProductName} supports the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.17/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#metro-dr-solution[Metro-DR solution for {rh-storage}], which provides two-way synchronous data replication between managed {VirtProductName} clusters installed on primary and secondary sites.
 
-Use this solution during a site disaster to failover applications from the primary to the secondary site, and relocate the applications back to the primary site after restoring the disaster site.
+.Metro-DR differences
+* This synchronous solution is only available to metropolitan distance data centers with a 10-millisecond latency or less. 
+* Multiple disk VMs are supported.
+* Storage is fenced during failover.
```

Comment:
@awels or @aglitke - do either of you have thoughts on how we could clarify this point? Thanks!

Reply:
Fencing is the process of isolating a node so that it won't run workloads. This prevents data from being corrupted. For instance with VMs, you don't want two VM processes accessing the same disk at the same time otherwise they would be writing over each other and corrupting the data. Fencing a node prevents workloads from on that node and we can guarantee that the workload is not running on that node when it is fenced.

Reply:
@awels - what does the bullet point want to tell me?
1. Storage is fenced by the solution
2. The customer needs to ensure the storage is fenced
3. Fencing is done under certain circumstances
4. Something else

Reply:
2. The customer needs to ensure the storage is fenced.

Reply:
This is a great clarification. Changing to the following for now; let me know if you have any concerns @mulbc - thanks!

```
* To prevent data corruption, you must ensure that storage is fenced during failover. 
+
[TIP]
====
Fencing means isolating a node so that workloads do not run on it. 
====
```